test/JDBC/expected/check_constraints_definition.out

```diff
@@ -0,0 +1,210 @@
+create type sch1.udd_datetime from sys.datetime;
+create type sch1.udd_time from time(6);
+go
+
+-- Inside check constraint definition alias datatypes are not allowed
+create table test_udd(
+	c_udd sch1.udd_float check(c_udd < 133.230182309832423),
+	c_udd1 sch1.udd_datetime check(c_udd1 < '2020-10-20 09:00:00'),
+	c_udd2 sch1.udd_time check(c_udd2 < '09:00:00'),
+);
+go
+
+create table test_null(a int, b int, check(a IS NOT NULL), CONSTRAINT constraint1 check (a>10));
+go
+
+create table test_upper(a char, check (upper(a) in ('A','B')));
+go
+
+alter table test_null add constraint constraint2 check(a<=20 and b>a and (a between 40 and 50));
+go
+
+set ansi_nulls on;
+go
+create table test_null1(a int, b int, check(a <> NULL), check(b = NULL));
+go
+	
+set ansi_nulls off;
+go
+
+create table test_null2(a int, b int, check(a <> NULL), check(b = NULL));
+go
+
+select * from information_schema_tsql.check_constraints order by "CONSTRAINT_NAME";
+go
+~~START~~
+nvarchar#!#nvarchar#!#varchar#!#nvarchar
+db1#!#dbo#!#constraint1test_null987b8e55c77de33d095778885ecd84f4#!#((a > 10))
+db1#!#dbo#!#constraint2test_nullba039d416ebe51c75b9d6d571d9a6f74#!#(((a <= 20) AND (b > a) AND ((a >= 40) AND (a <= 50))))
+db1#!#dbo#!#test_datetime_c_date_check#!#((c_date < CAST('2001-01-01' AS date)))
+db1#!#dbo#!#test_datetime_c_datetime2_check#!#(((c_datetime2 < CAST('2020-10-20 09:00:00' AS datetime2(6))) AND (c_datetime2 < CAST('2020-10-20 09:00:00' AS datetime2(6)))))
+db1#!#dbo#!#test_datetime_c_datetime_check#!#((c_datetime < CAST('2020-10-20 09:00:00' AS datetime)))
+db1#!#dbo#!#test_datetime_c_datetimeoffset_check#!#(((c_datetimeoffset < CAST('2025-12-10 12:32:10 +01:00' AS datetimeoffset)) AND (c_datetimeoffset < CAST('2025-12-10 12:32:10 +01:00' AS datetimeoffset(4)))))
+db1#!#dbo#!#test_datetime_c_smalldatetime_check#!#((c_smalldatetime < CAST('2007-05-08 12:35:00' AS smalldatetime)))
+db1#!#dbo#!#test_datetime_c_time_check#!#(((c_time < CAST('09:00:00' AS time)) AND (c_time < CAST('09:00:00' AS time(6)))))
+db1#!#dbo#!#test_functioncall_col1_check#!#((isjson(col1) > 0))
+db1#!#dbo#!#test_functioncall_col1_check1#!#(("right"(col1, 1) <> ','))
+db1#!#dbo#!#test_functioncall_col1_check2#!#((ltrim(col1) <> ''))
+db1#!#dbo#!#test_functioncall_col1_check3#!#((CAST((getutcdate() AT TIME ZONE col1) AS nvarchar(128)) <> CAST('' AS varchar)))
+db1#!#dbo#!#test_null1_a_check#!#((a <> CAST(NULL AS int)))
+db1#!#dbo#!#test_null1_b_check#!#((b = CAST(NULL AS int)))
+db1#!#dbo#!#test_null2_a_check#!#((a IS NOT NULL))
+db1#!#dbo#!#test_null2_b_check#!#((b IS NULL))
+db1#!#dbo#!#test_null_a_check#!#((a IS NOT NULL))
+db1#!#dbo#!#test_tsql_cast_c_decimal_check#!#(((c_decimal < CAST((CAST((133.23) AS money)) AS decimal(18,0))) AND (c_decimal < CAST((133.230182309832423) AS decimal(5,3)))))
+db1#!#dbo#!#test_tsql_cast_c_float_check#!#(((c_float < CAST((133.230182309832423) AS float)) AND (c_float < CAST((133.230182309832423) AS real))))
```

Comment:
We shall document such diffs like float(7) is being displayed as real.

Reply:
Added this as a comment in the test, will document it in a quip with more differences.